frontend/src/components/cfp-form/index.tsx

```diff
@@ -152,13 +135,8 @@ export const CfpForm: React.SFC<Props> = ({
           "tags",
           submission!.tags.map((t) => t.id),
         );
-        formState.setField(
-          "speakerLevel",
-          submission!.speakerLevel || SPEAKER_LEVEL_NEW_VALUE,
-        );
+        formState.setField("speakerLevel", submission!.speakerLevel);
```

Comment:
uhm we used to have the `||` because `speakerLevel` was a field added later, but I think now it is not needed anymore 👍